nasa-gibs/worldview
# Wv 3074 sub daily time slider

## Description

> Have Time Slider "Data Availability" Precision Match at Sub-Daily Scale

## How To Test

1. `git checkout WV-3074-sub-daily-time-slider`
2. `npm run watch`
3. Add a Granule Layer
4. The time slider may or may not have gaps based on the CMR API granule coverage.
5. The coverage bar(s) should match what is shown in the timeline
6. Other layers should behave normally

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
